### ISSUE TITLE
HMMLR - (Maybe) random crash

### ISSUE THREAD

**Issue by johnt28** [TNODevs/TNO-Bug-Tracker#607]
**Questions**

- Operating System: Windows 10
- HOI4 Version: 1.11.5
- TNO Version: 1.2.1.a
- Any other mods in use: No
- Were you using Steam? Yes
- Which expansions do you NOT have? I don't have the radio ones.

**Longer Description of Issue:**
Apparently, the game is crashing once Free France beat Cameroon African State in war, but being more wars happening in the world, that may or may not be the reason. While there is a CTD, this time it don't generate a log report specifically for that crash, so I will only send the save game.

**Steps to Reproduce/Cause Issue:**

1. Play until Free France win against Cameroon African State.

2.
[Ironman TNO Kingdom of England.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/8007344/Ironman.TNO.Kingdom.of.England.zip)

etc...

**Screenshots:**
[Ironman TNO Kingdom of England.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/8007346/Ironman.TNO.Kingdom.of.England.zip)

**Game Saves and Log Files, if possible:**

**Comment by tkgitno** [comment]
Known issue and hopefully fixed in dev build (Paradox and its terrible peace deals). Use this save If you want to continue the game
[Ironman TNO Kingdom of England.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/8007717/Ironman.TNO.Kingdom.of.England.zip)

**Comment by johnt28** [comment]
Thank you very much, but another thing to have in mind is that, I fixed myself with some instructions a crash with SLP in England, and the African War went another way in which France and all those lose against Cameroon African State, and crashed again. The new and weird thing about this is, that I have already played as the US and when that happened (Cameroon winning), everything worked as it should, so I found this new crash a bit strange.
Thanks again and keep up the amazing work.
